File: FootScout-Vue.WebAPI/Models/DTOs/ClubOfferCreateDTO.cs
namespace FootScout_Vue.WebAPI.Models.DTOs
{
    public class ClubOfferCreateDTO
    {
        public int PlayerAdvertisementId { get; set; }
        public int PlayerPositionId { get; set; }
        public string ClubName { get; set; }
        public string League { get; set; }
        public string Region { get; set; }
        public double Salary { get; set; }
        public string AdditionalInformation { get; set; }
        public string ClubMemberId { get; set; }
    }
}

File: FootScout-Vue.WebClient/src/models/dtos/ClubOfferCreateDTO.ts
// Modeluje ofertę klubu dla zawodnika, uwzględniając dane takie jak liga, region, 
// wynagrodzenie i dodatkowe informacje.

export interface ClubOfferCreateDTO {
    playerAdvertisementId: number;
    playerPositionId: number;
    clubName: string;
    league: string;
    region: string;
    salary: number;
    additionalInformation: string;
    clubMemberId: string;
}